packages/tailwindcss/src/utilities.ts

```diff
@@ -1929,6 +1965,10 @@ export function createUtilities(theme: Theme) {
   ])
 
   staticUtility('text-ellipsis', [['text-overflow', 'ellipsis']])
+
+  /** @deprecated */
+  staticUtility('overflow-ellipsis', [['text-overflow', 'ellipsis']])
```

Comment:
We should add this here: https://github.com/tailwindlabs/tailwindcss/blob/next/packages/tailwindcss/src/compat/legacy-utilities.ts

Reply:
ohhh I forgot we'd split these. Thanks for the reminder!